### DISCUSSION TITLE
feature request: integrate chia-exporter

### DISCUSSION THREAD

**Discussion by fisherwei** [guydavis/machinaris#952] [Ideas]
https://github.com/Chia-Network/chia-exporter

collect metrics to prometheus

**Comment by guydavis** [comment]
Please try [enabling this](https://github.com/guydavis/machinaris/wiki/Monitoring#chia-exporter) in the most recent `machinaris:develop` image.

**Reply by fisherwei** [reply]
you rock!

**Comment by fisherwei** [comment]
suggestion: chia_exporter runs in harvester mode too.

**Comment by guydavis** [comment]
Good enhancement. I've now added that launch to `machinaris:develop` image when `mode=harvester` as well.  Will be in the next release.
